addon-manager/tests/pom.xml

```diff
@@ -23,6 +23,12 @@
       </dependency>
       <dependency>
          <groupId>org.jboss.forge.addon</groupId>
+         <artifactId>ui-impl</artifactId>
```

Comment:
That would mean we need to improve the API in the UI addon. Exposing ui-impl is a bad idea